index.html

```diff
@@ -0,0 +1,226 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header id="header">
+        <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
+        <nav id="navbar">
+            <ul>
+                <li class="content-padding web"><a href="#works">Portfolio</a></li>
+                <li class="content-padding web"><a href="#about-myself">About</a></li>
+                <li class="content-padding web"><a href="#contact-form">Contact</a></li>
+                <li class="content-padding mobile"><a href="#"><img src="images/menu-icon.png" alt="menu"></a></li>
+            </ul>
+        </nav>
+        <div class="mobile-menu">
+            <img src="images/cross.svg" alt="cross">
+            <ul>
+                <li class=""><a href="#works">Portfolio</a></li>
+                <li class=""><a href="#about-myself">About</a></li>
+                <li class=""><a href="#contact-form">Contact</a></li>
+            </ul>
+        </div>
```

Comment:
- [ ] As the previous reviewer mentioned, I am still not able to close the mobile menu it is static and does not close even when I tap the close button. Kindly ensure that the mobile menu does not show by default and only shows when someone taps on the hamburger menu button.

**This is what I am seeing right now:**

[Screencast from 10-10-2022 09:07:16 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/194928087-a78b9f91-1a77-4152-8867-bd7fc50a315c.webm)